packages/formStructure/readme/fieldGroup.md

```diff
@@ -0,0 +1,7 @@
+# FieldGroup
+
+A `FieldGroup` may be used to visually group 2 or more fields that are closely related. For example: a number input followed by a select dropdown containing possible memory units.
```

Comment:
do you mean different unit selection? because sounds like from writing selecting a number 2x

Reply:
The first input is a number input, the second input is a <select> dropdown.

If this is unclear, I'm open to other examples.

Reply:
I see OK